test/lit.cfg

```diff
@@ -932,7 +932,8 @@ SIMULATOR_ENV_PREFIX = 'SIMCTL_CHILD_'
 ENV_VAR_PREFIXES = {
     'iphonesimulator': SIMULATOR_ENV_PREFIX,
     'watchsimulator': SIMULATOR_ENV_PREFIX,
-    'appletvsimulator': SIMULATOR_ENV_PREFIX
+    'appletvsimulator': SIMULATOR_ENV_PREFIX,
+    'android': 'ANDROID_CHILD_'
```

Comment:
Where does the prefix come from?  Is there something special about it?

Reply:
Nothing special, except that it has to match here and in the script that invokes the binary on the Android device. The contents are more or less following the iOS simulator one, but that’s not necessary.

Reply:
Oh, I see, rather than create a new variable, you just inlined it.  That makes sense.